### PR TITLE
*: fix govet -shadow in go tip

### PR DESCRIPTION
**Pull request by gyuho**

### CODE REVIEW

File: etcdserver/server.go [hunk 1]
```diff
@@ -296,7 +296,8 @@ func NewServer(cfg *ServerConfig) (srv *EtcdServer, err error) {
 			if err != nil {
 				return nil, &DiscoveryError{Op: "join", Err: err}
 			}
-			urlsmap, err := types.NewURLsMap(str)
+			urlsmap := make(types.URLsMap)
```
Comment (xiang90) [inline etcdserver/server.go]: var urlsmap types.URLsmap

Comment (gyuho) [inline etcdserver/server.go]: fixed. Thanks!

**Comment by heyitsanthony** [comment]
lgtm

**Comment by gyuho** [comment]
thanks.

**Pull request merged**